app/jobs/irs_attempts_events_batch_job.rb

```diff
@@ -42,4 +46,23 @@ def redis_client
   def s3_helper
     @s3_helper ||= JobHelpers::S3Helper.new
   end
+
+  def log_irs_attempts_events_job_metrix_info(result, events, start_time)
+    logger_info_hash(
+      name: 'IRSAttemptsEventJobMetrix',
+      start_time: start_time,
+      end_time: Time.zone.now,
+      duration_ms: duration_ms(start_time),
+      events_count: events.values.count,
+      file_bytes_size: result.encrypted_data.bytesize,
+    )
+  end
+
+  def logger_info_hash(hash)
+    logger.info(hash.to_json)
```

Comment:
Right now this will go to `production.log`, but we could use [workers.log](https://github.com/18F/identity-idp/blob/main/lib/identity_job_log_subscriber.rb#L124-L126) if that's more appropriate too.

Reply:
in my local, it is logging in workers.log file, i thin because of [this](https://github.com/18F/identity-idp/blob/main/lib/identity_job_log_subscriber.rb#L112)

Reply:
Ah sorry, you're right, it does go where it should (I think it's due to [this line](https://github.com/18F/identity-idp/blob/d326049b94bc4b31c3ee007cecc99684d94e99af/config/application.rb#L64))